src/views/Home.js

```diff
@@ -191,7 +190,7 @@ function Home() {
                         </Grid>
                     </Container>
                     <br />
-                    <Pagination count={1} /><br />{zoneContext !== undefined ? zoneContext.map((zone_report) =>
+                    <Pagination count={1} /><br />{zoneContext !== undefined ? 
```

Comment:
is there something missing here?

fine if not - just looks a tad strange.

Reply:
Ah, that's part of cleaning homepage process. Nothing is missing but should be included in the cleanup homepage commit.

Reply:
k, all good.